src/main/java/turniplabs/halplibe/helper/RegistryHelper.java

```diff
@@ -0,0 +1,19 @@
+package turniplabs.halplibe.helper;
+
+import java.util.ArrayList;
+
+public class RegistryHelper {
+	private static final ArrayList<Runnable> regsiterFunctions = new ArrayList<>();
```

Comment:
There is a typo here lol

Reply:
If you’re referring to "regsiterFunctions", I have a tendency to put variables that people aren’t supposed to access into different forms of the word I want to use